internal/cmd/stack/stack_selector.go

```diff
@@ -34,28 +34,28 @@ func getStackID(cliCtx *cli.Context) (string, error) {
 func getStack(cliCtx *cli.Context) (*stack, error) {
 	if cliCtx.IsSet(flagStackID.Name) {
 		stackID := cliCtx.String(flagStackID.Name)
-		stack, found, err := stackGetByID(cliCtx.Context, stackID)
+		stack, err := stackGetByID(cliCtx.Context, stackID)
 		if err != nil {
-			return nil, fmt.Errorf("failed to check if stack exists: %w", err)
-		}
+			if errors.Is(err, errNoStackFound) {
```

Comment:
Nit: You don't have to check if the error exists to call `errors.Is`. I prefer less nested indentation.

Reply:
64b2af1071b8f7f8c47dbf91688069beeba079d5